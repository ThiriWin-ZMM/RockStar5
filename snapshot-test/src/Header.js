import React from 'react';

const Header = props => (
    <div count={4}>
        <h1>Hello</h1>
        <p>This is snapshot tesing.</p>
    </div>
)

export default Header;