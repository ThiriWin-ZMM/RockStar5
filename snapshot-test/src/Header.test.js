import React from 'react'
import Header from './Header'
import renderer from 'react-test-renderer'

test('Header Component', () => {
    const tree = renderer.create(<Header count={2} />).toJSON();
    expect(tree).toMatchSnapshot();
})