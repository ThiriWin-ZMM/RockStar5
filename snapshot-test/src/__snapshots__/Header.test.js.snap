// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component 1`] = `
<div
  count={4}
>
  <h1>
    Hello
  </h1>
  <p>
    This is snapshot tesing.
  </p>
</div>
`;
